unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/WorldBlockerHandler.cs

```diff
@@ -0,0 +1,98 @@
+﻿using DCL.Configuration;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.Controllers
+{
+    public class WorldBlockerHandler : BlockerHandler
+    {
+        HashSet<Vector2Int> allLoadedParcelCoords = new HashSet<Vector2Int>();
+        HashSet<Vector2Int> blockersToRemove = new HashSet<Vector2Int>();
+        HashSet<Vector2Int> blockersToAdd = new HashSet<Vector2Int>();
+
+        static float milisecondsUsed = 0f;
+
+        public void SetupGlobalBlockers(Dictionary<string, ParcelScene> loadedScenes, float height, Transform parent)
```

Comment:
I think we can decouple this system from the `ParcelScene` class by just passing the parcels of the ready scenes from the caller, as I only we only are using `isReady` and `parcels` from it.

Reply:
Done